scheduler/test/cook/test/simulator.clj

```diff
@@ -325,12 +340,42 @@
                                                  :straggler-trigger-chan (async/chan)
                                                  :lingering-task-trigger-chan (async/chan)
                                                  :cancelled-task-trigger-chan (async/chan)}})
-        {:keys [time-ms-between-optimizer-calls time-ms-between-rebalancing]} config]
+        {:keys [time-ms-between-incremental-output time-ms-between-optimizer-calls time-ms-between-rebalancing]} config]
+    ;; launch a thread to perform incremental writes of completed jobs
+    (when temp-out-trace-file
+      (let [last-start-time-atom (atom simulation-time)
+            write-thread (Thread.
+                           ^Runnable
+                           (fn write-completed-tasks []
+                             (while true
+                               (Thread/sleep time-ms-between-incremental-output)
+                               (try
+                                 (let [start-time @last-start-time-atom
+                                       current-time (.getMillis (t/now))
+                                       _ (log/info "Starting temporary output iteration" start-time "to" current-time)
+                                       start-date (Date. ^long start-time)
+                                       end-date (Date. ^long current-time)
+                                       completed-task-ents (pull-all-task-ents-completed-in-time-range
+                                                             (d/db mesos-datomic-conn) start-date end-date)]
+                                   (if (seq completed-task-ents)
+                                     (do
+                                       (log/info "Writing" (count completed-task-ents) "tasks to temporary output for simulation time" current-time)
+                                       (dump-jobs-to-csv completed-task-ents temp-out-trace-file
+                                                         :add-headers (= start-time simulation-time)
+                                                         :append true))
+                                     (log/info "No completed tasks in last" time-ms-between-incremental-output "ms"))
+                                   (reset! last-start-time-atom current-time))
+                                 (catch Throwable ex
+                                   (log/error ex "Error writing incremental completed job output!"))))))]
+        (log/info "temporary output will be updated every" time-ms-between-incremental-output "ms")
```

Comment:
The other log lines start with a capitalized word, but this one doesn't

Reply:
Changed to `Temporary`